src/machines/modelingMachine.ts

```
@@ -2866,6 +2870,92 @@ export const modelingMachine = setup({
         )
       }
     ),
+    cloneAstMod: fromPromise(
+      async ({
+        input,
+      }: {
+        input: ModelingCommandSchema['Clone'] | undefined
+      }) => {
+        if (!input) return new Error('No input provided')
```

Should probably reject here?

---

Alright these are super valid. Just now I specifically tried to mess around with cases that would trigger that it it would fail silently. Turned out I needed two things to make the errors show up:
1. change from `return new Error(...` to `return Promise.reject(new Error(...` as you suggested
2. change `'Applying clone'`'s `onError` from `onDone: ['idle']` to `onError: { target: 'idle', actions: 'toastError' }`, which of course makes sense!
 
Pushed these changes here, and I'll create a follow up issue to make some of the other codmods that are likely to have silent errors have those changes as well

---

https://github.com/KittyCAD/modeling-app/issues/6514